src/XMLimport.cpp

```diff
@@ -829,6 +829,16 @@ void XMLimport::readHostPackage(Host* pHost)
         pHost->mSearchEngineName = QString("Google");
     }
 
+    if (attributes().hasAttribute(QLatin1String("MinimumTimeForTimerDebugOutput"))) {
+        pHost->mTimerDebugOutputSuppressionInterval = QTime::fromString(attributes().value(QLatin1String("MinimumTimeForTimerDebugOutput")).toString(), QLatin1String("hh:mm:ss.zzz"));
+    } else {
+        // Ensure a null (zero) interval is used if none is stored (although the
+        // Host constructor does default to a zero value - if a previous profile
+        // save had set a different value and then another profile was read
+        // without the setting then it would not get reset in that case):
```

Comment:
This comment is more confusing than helpful... what is it trying to actually say?

Reply:
If the particular XML file being read does not contain this attribute (i.e. it predates the feature) we should ensure the Profile gets the setting reset to zero; the starting value in the `Host` constructor will cause this i.e. the value will be a null (a.k.a. a `h:mm:ss.zzz` value equal to `0:00:00.000`) but if another profile file has been read *first* **with** a value before *this* file is read, then that setting will not get replaced otherwise...

Does that make thing any clearer?

Reply:
Would it be? Are other profile loads actually affected by previous ones?

Reply:
Humm, perhaps I have grabbed the wrong end of the stick after all - once a profile is chosen from the Connection Preferences and a particular save file is selected it does not look as though a different one can be loaded afterwards - this contrasts with the map file loading where a new one can be reloaded over an old one... :open_mouth:

Reply:
In hindsight, the comment is redundant - I'll nuke it...